python/paddle/fluid/param_attr.py

```diff
@@ -65,14 +65,16 @@ def __init__(self,
                  regularizer=None,
                  trainable=True,
                  gradient_clip=None,
-                 do_model_average=False):
+                 do_model_average=False,
+                 distributed=False):
```

Comment:
solution is found, so that we will remove this attribute from ParamAttr. And as it is in the Distributed FC domain, we will handle this parameter in the next pr.